pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -80,7 +80,8 @@ func GetTaskFuncFromTaskRun(ctx context.Context, k8s kubernetes.Interface, tekto
 // a remote image to fetch the  reference. It will also return the "kind" of the task being referenced.
 // OCI bundle and remote resolution tasks will be verified by trusted resources if the feature is enabled
 func GetTaskFunc(ctx context.Context, k8s kubernetes.Interface, tekton clientset.Interface, requester remoteresource.Requester,
-	owner kmeta.OwnerRefable, tr *v1.TaskRef, trName string, namespace, saName string, verificationPolicies []*v1alpha1.VerificationPolicy) GetTask {
+	owner kmeta.OwnerRefable, tr *v1.TaskRef, trName string, namespace, saName string, verificationPolicies []*v1alpha1.VerificationPolicy,
+) GetTask {
```

Comment:
this extra comma `,` seems like typo

Reply:
It's probably the formatter (`gofumpt` I think)  that automatically does that. It is valid in go btw (otherwise it wouldn't compile), essentially, when you have a "list" of things separated by a `,` in multiple line, you *always* have a `,` at the end, even if it's the last item. And `gofumpt` force, in that case, the `)` on a new line as well.

Reply:
sorry ignore this, new line is added